test/features/homepage.feature

```diff
@@ -1,6 +0,0 @@
-@smoke
```

Comment:
do not remove this test! This is smoke test placed here on purpose. It is used to make sure that all systems work correctly. Please reinstate it.